mlrun/projects/project.py

```diff
@@ -1797,28 +1797,126 @@ def set_model_monitoring_application(
         **application_kwargs,
     ) -> mlrun.runtimes.BaseRuntime:
         """
-        update or add a monitoring application to the project & deploy it.
+        update or add a monitoring function to the project.
+
         examples::
-            project.set_model_monitoring_application(application_class_name="MyApp",
-                                                 image="davesh0812/mlrun-api:1.5.0", name="myApp")
-        :param func:                    function object or spec/code url, None refers to current Notebook
-        :param name:                    name of the function (under the project), can be specified with a tag to support
+            project.set_model_monitoring_function(application_class_name="MyApp",
+                                                 image="mlrun/mlrun-api:1.5.0", name="myApp")
```

Comment:
why are you using mlrun-api? this image should not be used by users. if you can't use mlrun/mlrun or some other user facing image, we should create a dedicated image.
also is it available in mlrun 1.5.0? because I see you set the tag to 1.5.0 but this will be released in 1.6.0